### PR TITLE
CLDR-15918 Add raj, bgc, bho, ti values required for Basic coverage

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-15918

- [x] This PR completes the ticket.

### CODE REVIEW

**Review by pedberg-icu — DISMISSED** [review]

**Review by srl295 — DISMISSED** [review]
ticket not accepted?

**Comment by macchiati** [comment]
Annemarie, would you mind adding the 4th item? 
https://raw.githubusercontent.com/unicode-org/cldr-staging/main/docs/charts/42/tsv/locale-missing.tsv#:~:text=ti%09Tigrinya%09Ethiopic%09basic%09basic%09n/a%09null%09Timezones%09Europe%09GMT
That is, copying "GMT" or setting to inherit?

**Comment by AEApple** [comment]
> Yes, I will update it later today when I have a moment.

**Comment by macchiati** [comment]
If you see any obvious (&easy!) ones in that tsv, we could include them too.

**Comment by AEApple** [comment]
I'm still following up on this one. Sorry for the delay!

**Comment by macchiati** [comment]
no problem

**Comment by AEApple** [comment]
> Annemarie, would you mind adding the 4th item? https://raw.githubusercontent.com/unicode-org/cldr-staging/main/docs/charts/42/tsv/locale-missing.tsv#:~:text=ti%09Tigrinya%09Ethiopic%09basic%09basic%09n/a%09null%09Timezones%09Europe%09GMT That is, copying "GMT" or setting to inherit?

Added

**Comment by AEApple** [comment]
Not sure why the addition of the metazone name for GMT in ti is causing a failure.

File: common/main/ti.xml [hunk 1]
```diff
@@ -3944,6 +3944,10 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<long>
 					<standard>ግዜ ጋላፓጎስ</standard>
 				</long>
+			<metazone type="GMT">
+				<long>
+					<standard>GMT</standard>
+				</long>
```
Comment (srl295) [inline common/main/ti.xml:3950]:
```suggestion
				</long>
		 </metazone>
```

missing end tag

File: common/main/ti.xml [hunk 2]
```diff
@@ -3944,6 +3944,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<long>
 					<standard>ግዜ ጋላፓጎስ</standard>
 				</long>
+			<metazone type="GMT">
```
Comment (srl295) [inline common/main/ti.xml:3947]:
```suggestion
			</metazone>
			<metazone type="GMT">
```

Comment (srl295) [reply common/main/ti.xml:3947]: i missed this one as well

**Review by macchiati — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

**Pull request merged**